go/libkb/social_assertion.go

```diff
@@ -8,20 +8,12 @@ import "strings"
 // SocialAssertion is a type for individual user's on a social
 // network.  It should be created via NormalizeSocialAssertion.
 type SocialAssertion struct {
-	username string
-	service  string
-}
-
-func (s SocialAssertion) Username() string {
-	return s.username
-}
-
-func (s SocialAssertion) Service() string {
-	return s.service
+	Username string
+	Service  string
```

Comment:
General question: we don't foresee this struct ever getting additional members later, right? (Otherwise, we may want to embed codec.UnknownFieldSet.)